dashboard/dashboard.py

```diff
@@ -37,14 +38,15 @@ class Dashboard:
 
     def __init__(
         self,
-        host,
-        port,
-        port_retries,
-        gcs_address,
-        log_dir=None,
-        temp_dir=None,
-        session_dir=None,
-        minimal=False,
+        host: str,
+        port: int,
+        port_retries: int,
+        gcs_address: str,
+        log_dir: str = None,
+        temp_dir: str = None,
+        session_dir: str = None,
+        minimal: bool = False,
+        modules_to_load: Optional[Set[str]] = None,
```

Comment:
What if minimal conflicts with modules_to_load?

Could you also remind me why do we need to pass in the `--minimal` flag instead of figuring out by itself inside dashboard process?